### DISCUSSION TITLE
啥时能修复闪退的问题啊？

### DISCUSSION THREAD

**Discussion by Evan7411** [getlantern/lantern-forum-cn#375] [General]
系统是MacOS. 如果修复好了能提醒下吗？

**Comment by MaxMKMax** [comment]
@Evan7411 您好! 嗯嗯. 我们知道闪退的问题. 7.8.4 今晚或明天会放出来了, 您等我发布通知下载最新版本. 谢谢您的耐心等待!

**Reply by studybioMR01** [reply]
修复完成麻烦提醒一下，等了好几天了

**Reply by MaxMKMax** [reply]
@studybioMR01 好的好的. 我会弄个通告通知. 谢谢您的等待.
但是现有版本他们好像也修复了. 您试试看 7.8.3 还有什么问题吗?

**Comment by studybioMR01** [comment]
并没有找到7.8.3的版本



---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 日期 | 2024年04月04日 02:42 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 啥时能修复闪退的问题啊？ (Discussion #375) |

@studybioMR01 好的好的. 我会弄个通告通知. 谢谢您的等待.
但是现有版本他们好像也修复了. 您试试看 7.8.3 还有什么问题吗?

**Reply by MaxMKMax** [reply]
@studybioMR01 
7.8.4
https://github.com/getlantern/lantern/blob/devel/README.zh.md

https://github.com/getlantern/lantern

**Comment by MaxMKMax** [comment]
@studybioMR01 @Evan7411 
各位好! 这两个连接都可以下载. 这个版本应该修复了闪退和别的问题了. 如果还有什么问题, 请再来反馈.
或者有些用户已自动更新了! 请查看!
谢谢你们耐心的等待!

https://github.com/getlantern/lantern/blob/devel/README.zh.md

https://github.com/getlantern/lantern

**Reply by studybioMR01** [reply]
我现在没办法自动回到我原来购买的专业版

**Reply by studybioMR01** [reply]
而之前用的邮箱也收不到验证码

**Reply by MaxMKMax** [reply]
@studybioMR01 
请您提供我您的账号邮箱. 我来帮您找付款部门要激活码.

**Reply by studybioMR01** [reply]
shangyang3387@163.com

**Reply by MaxMKMax** [reply]
@studybioMR01 好的谢谢您. 我帮您提交上去.

**Reply by MaxMKMax** [reply]
@studybioMR01 
你好。我们的系统团队需要您的付款收据以及您购买了什么计划？请帮忙提供给我。谢谢。

**Comment by studybioMR01** [comment]
我是通过支付宝购买的



---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 日期 | 2024年04月06日 02:31 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 啥时能修复闪退的问题啊？ (Discussion #375) |

@studybioMR01
你好。我们的系统团队需要您的付款收据以及您购买了什么计划？请帮忙提供给我。谢谢。

**Comment by studybioMR01** [comment]
你好，什么时候可以授权



---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 日期 | 2024年04月06日 02:31 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 啥时能修复闪退的问题啊？ (Discussion #375) |

@studybioMR01
你好。我们的系统团队需要您的付款收据以及您购买了什么计划？请帮忙提供给我。谢谢。

**Comment by studybioMR01** [comment]
你好，为什么一直不给我授权



---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 日期 | 2024年04月06日 02:31 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 啥时能修复闪退的问题啊？ (Discussion #375) |

@studybioMR01
你好。我们的系统团队需要您的付款收据以及您购买了什么计划？请帮忙提供给我。谢谢。

**Reply by MaxMKMax** [reply]
@studybioMR01 您好. 支付团队需要您的购买单据常能提供您授权码. 因为您的购买记录没在系统里. 估计系统有问题. 支付团队让我来向您获取支付单据的. 谢谢您!

**Comment by studybioMR01** [comment]
支付单据上次不是给你了吗



---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 日期 | 2024年04月14日 22:39 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 啥时能修复闪退的问题啊？ (Discussion #375) |

@studybioMR01 您好. 支付团队需要您的购买单据常能提供您授权码. 因为您的购买记录没在系统里. 估计系统有问题. 支付团队让我来向您获取支付单据的. 谢谢您!
